packages/product-components/src/components/TokenSummaryIcon/TokenSummaryIcon.tsx

```diff
@@ -0,0 +1,62 @@
+import { AssetLogo, useElevation } from '@trezor/components';
+import { getCoingeckoId } from '@suite-common/wallet-config';
+import { EnhancedTokenInfo, getContractAddressForNetwork } from '@suite-common/token-definitions';
+import { NetworkSymbol } from '@suite-common/wallet-config';
+
+import styled, { css } from 'styled-components';
+import { borders, Elevation, mapElevationToBackground, mapElevationToBorder } from '@trezor/theme';
+
+export type TokenSummaryIconProps = {
+    network: NetworkSymbol;
+    tokens: EnhancedTokenInfo[];
```

Comment:
this component does not need to know it is `EnhancedTokenInfo`,  `TokenInfo` is enough I think